lib/mayaUsd/ufe/UsdUndoRenameCommand.cpp

```diff
@@ -91,45 +92,45 @@ UsdSceneItem::Ptr UsdUndoRenameCommand::renamedItem() const
 bool UsdUndoRenameCommand::renameRedo()
 {
     auto primSpec = MayaUsdUtils::getPrimSpecAtEditTarget(_stage, _prim);
-
-    if(primSpec) {
-
-        // set prim's name
-        primSpec->SetName(_newName);
-
-         // the renamed scene item is a "sibling" of its original name.
-        _ufeDstItem = createSiblingSceneItem(_ufeSrcItem->path(), _newName);
-
-        sendRenameNotification(_ufeDstItem, _ufeSrcItem->path());
+    if(!primSpec) {
+        return false;
     }
-    else {
+
+    // set prim's name
+    // XXX: SetName successfuly returns true but when you examine the _prim.GetName()
+    // after the rename, the prim name shows the original name HS, 6-May-2020.
+    bool status = primSpec->SetName(_newName);
+    if (!status) {
         return false;
     }
```

Comment:
I do an early check for primSpec and SetName result. 

@ppt-adsk  there is something troubling happening here which I am trying to understand:

On Redo, I get the primspec for _prim object and rename it's name to the _newName. This operation return successful BUT when I query the _prim's name right after, I still get the _prim's original name. Huh???

I then set _ufeDstItem and when I print it's USD name and it gives me the _newName which is expected.